apostrophecms/apostrophe-forms-google-address-field-widgets
# Adds labels to displayed split address fields

* Jira: [IN-14](https://apostrophecms.atlassian.net/browse/IN-14)
* Gitlab: [1071](https://gitlab-dcadcx.michelin.net/pa/apostrophe-enhancements/-/issues/1071)

## Purpose
Adding labels to splitted fields directly, to avoid having to modify translation files.

## How to test

For Migration do this with old apostrophe-forms-google-address-field-widgets version (or one existing)

* Activate `apostrophe-forms` modules in your project:
```
'apostrophe-forms': {
      formWidgets: {
        'apostrophe-forms-google-address-field': {}
      }
    },
    'apostrophe-forms-widgets': {},
    'apostrophe-forms-google-address-field-widgets': {
      googleApiKey: 'validGoogleApiKeyHere' // mandatory
    }
```
* Go to Forms in admin bar
* Create a Form
* Go to Form tab
* Ceate a Google Address
* Go to Advances Settings tab
* Split Address: Yes

* Activate new version of apostrophe-forms-google-address-field-widgets
* relaunch project, and verify that the form still works, then
* Fill the different fields with their labels
* Display split address: Yes
* Check that your form use the edited labels
